src/multiplayer-servers/multiplayer-services/server-allocation/automatically-registering-game-servers.md

```diff
@@ -168,6 +168,22 @@ which can only be achieved with a shared volume, mounted into both containers.
 This has to be done manually.
 :::
 
+### Return a payload to the Allocator
+
+In order to return information about the game server to the process calling `/allocate` 
+[add the environment variable `ALLOC_CALLBACK_PAYLOAD_ANNOTATION`](#alloc-callback-payload-annotation-string) on the Allocation Sidecar container.
+The Allocation Sidecar will read the annotations of the game server, and those prefixed with the given string will be compiled
+into a payload that will be sent to the Allocator when the game server is allocated.
```

Comment:
```suggestion
The Allocation Sidecar reads game server annotations, and those prefixed with the given string get compiled
into a payload that is sent to the Allocator when the game server is allocated.
```

Always clearer to use general truth than future IMO. Here since we end the sentence with "When the game server is allocated" that answers the questions that future tense could raise, so it's not a blocker imo, but it still induces potential confusion in the first parts of the sentence, and should be avoided in general unless there is no alternative (i.e. when we talk about something that is not true today, but will be true in the actual future)